doc/topics/domain-postmortem.md

```diff
@@ -109,7 +109,7 @@ leverage the implicit mechanisms already in place.
 
 Propagating errors across nested domains is not straight forward, if even
 possible. Existing documentation shows a simple example of how to `close()` an
-`http` server if there is an error in the request handler. What it does not
+`HTTP` server if there is an error in the request handler. What it does not
```

Comment:
I disagree with this. Typically when backticks are involved, it can be referring to the built-in module of that name. Although in this particular case, a better change *might* be to make '`http` server' instead read '`http.Server`'.